buildtools/common.props

```diff
@@ -13,7 +13,7 @@
 	
     <PackageIconUrl>https://sdk-for-net.amazonwebservices.com/images/AWSLogo128x128.png</PackageIconUrl>
     <PackageProjectUrl>https://github.com/aws/aws-lambda-dotnet</PackageProjectUrl>
-    <PackageLicenseUrl>http://aws.amazon.com/apache2.0/</PackageLicenseUrl>
+    <PackageLicenseUrl>https://aws.amazon.com/apache2.0/</PackageLicenseUrl>
```

Comment:
Since we are making changes can we change this to `PackageLicenseExpression` like we do in the messaging package. `PackageLicenseUrl` is deprecated and we can get rid of a lot of warnings in our build by moving away from `PackageLicenseUrl`.
```
<PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
```